mlflow/ml_package_versions.py

```diff
@@ -297,7 +297,7 @@
         },
         "autologging": {
             "minimum": "0.2.36",
-            "maximum": "0.2.39"
+            "maximum": "0.4.0"
```

Comment:
```suggestion
            "maximum": "0.2.40"
```
We don't officially support latest autogen (after the major package break up)